jbpm-event-emitters/jbpm-event-emitters-elasticsearch/src/main/java/org/jbpm/event/emitters/elasticsearch/ElasticSearchEventEmitter.java

```diff
@@ -135,7 +135,7 @@ public void apply(Collection<InstanceView<?>> data) {
             try {
                 HttpPut httpPut = new HttpPut(elasticSearchUrl + "/_bulk");
                 httpPut.setEntity(new StringEntity(content.toString(), "UTF-8"));
-                logger.debug("Elastic search request {}", httpPut.getRequestLine());
```

Comment:
Is this needed?
I mean Im assuming, maybe incorrectly, that getRequestLine include everthing, headers and context, is not that the case?

Reply:
The output of httpPut.getRequestLine() looks like this:
DEBUG [org.jbpm.event.emitters.elasticsearch.ElasticSearchEventEmitter] (pool-18-thread-1) Elastic search request PUT http://localhost:9200/_bulk HTTP/1.1

It does not really help to troubleshoot any issues with the request payload.